chef.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "mixlib-cli", "~> 1.4"
   s.add_dependency "mixlib-log", "~> 1.3"
-  s.add_dependency "mixlib-authentication", "~> 1.3"
+  s.add_dependency "mixlib-authentication", ">= 1.4.0.rc.1", "< 1.5"
```

Comment:
1.4.0 went out on January 19, 2016 so we should bump the dependency to the stable release here now.